src/site/layouts/home.drupal.liquid

```diff
@@ -0,0 +1,139 @@
+{% include "src/site/includes/header.html" %}
+<main>
+
+  <!-- the hub -->
+  <section class="homepage-hub">
+
+    <div class="homepage-hub-container">
+
+      <h1 class="heading-level-2 homepage-heading">Access and manage your VA benefits and health care</h1>
+
+      <!-- top row -->
+      <div class="hub-links-row">
+      {% for card in cards %}
+        <div class="hub-links-container">
+          <h2 class="heading-level-3 hub-links-title"><i class="icon-large-baseline icon-heading hub-icon-{{ card.label | downcase | replace: ' ', '-' }} hub-color-{{ card.label | downcase | replace: ' ', '-' }}"></i>{{ card.label }}</h2>
+          <ul class="hub-links-list">
+            {% assign parentCard = card %}
```

Comment:
I know this is in `home.html` before, but I think it might be vestigial. Can we get rid of `parentCard` altogether?